__tests__/app.test.js

```diff
@@ -56,46 +56,91 @@ describe("NC News Server", () => {
   });
   describe("POST CUSTOM ERRORS", () => {
     test("POST 404 : returns error if username does not exist", () => {
-        const newComment = { username: 'i am not real', body: 'body of comment 123' }
-        return request(app)
-            .post('/api/articles/1/comments')
-            .send(newComment)
-            .expect(404)
-            .then(({ body }) => {
-            expect(body.message).toBe("User Not Found");
-          });
-    })
+      const newComment = {
+        username: "i am not real",
+        body: "body of comment 123",
+      };
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send(newComment)
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.message).toBe("User Not Found");
+        });
+    });
     test("POST 400 : /api/articles/:article_id/comments with missing body of comment", () => {
-        const newComment = { username: 'lurker' }
-        return request(app)
-            .post('/api/articles/1/comments')
-            .send(newComment)
-            .expect(400)
-            .then(({ body}) => {
-            expect(body.message).toBe("Bad Request");
-          });
-      });
-      test("POST 400 : /api/articles/:article_id/comments when article_id is wrong format comment", () => {
-        const newComment = { username: 'lurker', body: 'body of comment 123' }
-        return request(app)
-            .post('/api/articles/wrongformat/comments')
-            .send(newComment)
-            .expect(400)
-            .then(({ body}) => {
-            expect(body.message).toBe("Bad Request");
-          });
-      });
-      test("POST 404 : /api/articles/:article_id/comments when posting on article that does not exist", () => {
-        const newComment = { username: 'lurker', body: 'body of comment 123' }
-        return request(app)
-            .post('/api/articles/1000/comments')
-            .send(newComment)
-            .expect(404)
-            .then(({ body}) => {
-            expect(body.message).toBe("Article Not Found");
-          });
-      });
-  })
+      const newComment = { username: "lurker" };
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send(newComment)
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.message).toBe("Bad Request : PSQL Input error");
+        });
+    });
+    test("POST 400 : /api/articles/:article_id/comments when article_id is wrong format comment", () => {
```

Comment:
I'd slightly clarify this to wrong data type / bad request because format has a slightly different meaning